### PR TITLE
PROVIDERS: AzureBlobStorage ListFilesInContainer

### PR DESCRIPTION
**Pull request by MichaelOSullivanAnswer**
CLOSES #AB17832

### CODE REVIEW

File: ISL.Providers.Storages.AzureBlobStorage.Tests.Unit/Providers/AzureBlobStorage/AzureBlobStorageTestsExceptions.ListFilesInContainer.cs [hunk 1]
```diff
@@ -0,0 +1,177 @@
...
+            string inputContainer = randomContainer;
+
+            var storageDependencyException = new StorageDependencyException(
+                message: "Storage dependency error occurred, please fix errors and try again.",
+                innerException: new Xeption());
+
+            var expectedAzureBlobStorageProviderDependencyException =
+                new AzureBlobStorageProviderDependencyException(
+                    message: "Azure blob storage provider dependency error occurred, " +
+                            "contact support.",
```
Comment (cjdutoit) [inline ISL.Providers.Storages.AzureBlobStorage.Tests.Unit/Providers/AzureBlobStorage/AzureBlobStorageTestsExceptions.ListFilesInContainer.cs:109]: fix indentation

File: ISL.Providers.Storages.AzureBlobStorage.Tests.Unit/Providers/AzureBlobStorage/AzureBlobStorageTestsExceptions.ListFilesInContainer.cs [hunk 2]
```diff
@@ -0,0 +1,177 @@
...
+            string inputContainer = randomContainer;
+
+            var storageDependencyValidationException = new StorageDependencyValidationException(
+                message: "Storage dependency validation error occurred, please fix errors and try again.",
+                innerException: new Xeption());
+
+            var expectedAzureBlobStorageProviderDependencyValidationException =
+                new AzureBlobStorageProviderDependencyValidationException(
+                    message: "Azure blob storage provider dependency validation error occurred, " +
+                            "fix errors and try again.",
```
Comment (cjdutoit) [inline ISL.Providers.Storages.AzureBlobStorage.Tests.Unit/Providers/AzureBlobStorage/AzureBlobStorageTestsExceptions.ListFilesInContainer.cs:67]: fix indentation:

```suggestion
                    message: "Azure blob storage provider dependency validation error occurred, " +
                        "fix errors and try again.",
```

OR

```suggestion
                    message: 
                        "Azure blob storage provider dependency validation error occurred, " +
                        "fix errors and try again.",
```

File: ISL.Providers.Storages.AzureBlobStorage.Tests.Unit/Providers/AzureBlobStorage/AzureBlobStorageTestsExceptions.ListFilesInContainer.cs [hunk 3]
```diff
@@ -0,0 +1,177 @@
...
+            string inputContainer = randomContainer;
+
+            var storageServiceException = new StorageServiceException(
+                message: "Storage service error occurred, please fix errors and try again.",
+                innerException: new Xeption());
+
+            var expectedAzureBlobStorageProviderServiceException =
+                new AzureBlobStorageProviderServiceException(
+                    message: "Azure blob storage provider service error occurred, " +
+                            "contact support.",
```
Comment (cjdutoit) [inline ISL.Providers.Storages.AzureBlobStorage.Tests.Unit/Providers/AzureBlobStorage/AzureBlobStorageTestsExceptions.ListFilesInContainer.cs:150]: fix

File: ISL.Providers.Storages.AzureBlobStorage.Tests.Unit/Providers/AzureBlobStorage/AzureBlobStorageTestsExceptions.ListFilesInContainer.cs (hunk 1 above)
Comment (MichaelOSullivanAnswer) [reply ISL.Providers.Storages.AzureBlobStorage.Tests.Unit/Providers/AzureBlobStorage/AzureBlobStorageTestsExceptions.ListFilesInContainer.cs]: this has been updated

File: ISL.Providers.Storages.AzureBlobStorage.Tests.Unit/Providers/AzureBlobStorage/AzureBlobStorageTestsExceptions.ListFilesInContainer.cs (hunk 2 above)
Comment (MichaelOSullivanAnswer) [reply ISL.Providers.Storages.AzureBlobStorage.Tests.Unit/Providers/AzureBlobStorage/AzureBlobStorageTestsExceptions.ListFilesInContainer.cs]: this has been updated

File: ISL.Providers.Storages.AzureBlobStorage.Tests.Unit/Providers/AzureBlobStorage/AzureBlobStorageTestsExceptions.ListFilesInContainer.cs (hunk 3 above)
Comment (MichaelOSullivanAnswer) [reply ISL.Providers.Storages.AzureBlobStorage.Tests.Unit/Providers/AzureBlobStorage/AzureBlobStorageTestsExceptions.ListFilesInContainer.cs]: this has been updated

**Review by cjdutoit — APPROVED** [review]

**Pull request merged**